### DISCUSSION TITLE
Get all path parameters from a route object

### DISCUSSION THREAD

**Discussion by veerack** [fastapi/fastapi#6445] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get('/api/docs')
async def api_endpoints(request: Request):
    d = {
        "endpoints": [{
            "params": route.parameters
        } for route in api_router.routes]
    }
    return d
```

### Description

Trying to get all the path parameters of each route object, but apparently `route.parameters` isn't a thing..
I researched on google for hours but i didn't find anything related to this.

My goal should be able to get all the route parameters, and the output like
```
{'endpoints': [{'params': ['param_1 int', 'param_2 str']}]}
```

### Operating System

Linux

### FastAPI Version

lastest one

### Python Version

3.9.4

**Comment by poximy** [comment, marked as answer]
Path parameters can be accessed in `request.path_params`, this will return a dictionary
